.github/workflows/age-ci.yml

```diff
@@ -3,7 +3,7 @@ name: Age CI
 on:
   push:
     paths:
-      - age/**
+      - age/contracts/**
```

Comment:
Why ? Don't we want to rerun it if the front change ?

Also true for the others.

Reply:
Because these tests start with the contract file

Reply:
But, I follow your instruction if you prefer to do it

Reply:
done